consoles/VNC.pm

```diff
@@ -21,7 +20,7 @@ __PACKAGE__->mk_accessors(
     qw(hostname port username password socket name width height depth save_bandwidth
       no_endian_conversion  _pixinfo _colourmap _framebuffer _rfb_version screen_on
       _bpp _true_colour _do_endian_conversion absolute ikvm keymap _last_update_received
-      _last_update_requested _vnc_stalled vncinfo old_ikvm
+      _last_update_requested _vnc_stalled vncinfo old_ikvm dell
```

Comment:
Quickly going though the changes and I don't see anywhere this variable is set. Found only where is checked. Is it intentional?